.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+Thank you for reporting an issue with **AlexaPi**.
+
+Make sure you are running the latest version of **AlexaPi** before reporting an issue.
+
+Please go through known issues before submitting:
+    - Open: https://github.com/alexa-pi/AlexaPi/issues?q=is%3Aopen+is%3Aissue
+    - Closed: https://github.com/alexa-pi/AlexaPi/issues?q=is%3Aissue+is%3Aclosed
+    
+You should only file an issue if you found a bug or are stuck getting everything configured properly.
+
+Our wiki may be found at https://github.com/alexa-pi/AlexaPi/wiki
```

Comment:
From this line up to above _**Python release (`python2 --version`):**_ this info should be only in the wiki and there should be only links. Make the links a text, not URL. This is way too long.